app/gui2/src/components/ComponentBrowser.vue

```diff
@@ -316,6 +318,10 @@ const handler = componentBrowserBindings.handler({
     applySuggestion()
   },
   acceptInput() {
+    applySuggestion()
+    acceptInput()
```

Comment:
```suggestion
    applySuggestion()
```
Applying suggestion should also accept input, unless we want to enter module, in which case CB should _not_ close.

Also, I would rename it as `acceptSuggestion`, and `acceptInputRaw` could be just `acceptInput`.

Reply:
The `applySuggestion` method, right now, adds the currently selected suggestion in the CB to the input. This is what happens when pressing `tab`.

I thought we had three interactions here: 
* `tab` only applies the selected suggestion but keeps the CB open
* `enter` chooses the applied the selected suggestion and closes the CB 
* `ctrl+enter` closes the CB with the current input, without applying the selected suggestion.

Reply:
Sorry, I re-read my comment, and now I cannot say what have I had in mind. You are right, of course. Only the names:
* `tab` I usually name "applying suggestion"
* `enter` is "_accepting_ suggestion"
* `ctrl+enter` is "accepting _input_"

Reply:
That sounds good, I'll apply that naming to the code, add a doc and that should make it consistent and clear.